### ISSUE TITLE
Support rails HEAD (5.0.0 as will be)

### ISSUE THREAD

**Issue by caius** [fac/hestia#9]
https://github.com/fac/hestia/compare/rails-head starts this work, currently has a failing spec (& depends on the rails-4.2 branch being merged first.)

```
Thrax:hestia(rails-head) caius$ BUNDLE_GEMFILE=Gemfile.railshead bundle exec rake spec
Run options: --seed 40513

# Running:

...................E.

Fabulous run in 0.004255s, 4935.3168 runs/s, 7285.4677 assertions/s.

  1) Error:
Hestia::MessageMultiVerifier::verify behaviour#test_0008_verifies a message of `nil' successfully:
ActiveSupport::MessageVerifier::InvalidSignature: ActiveSupport::MessageVerifier::InvalidSignature
    /Users/caius/freeagent/src/hestia/lib/hestia/message_multi_verifier.rb:57:in `verify'
    /Users/caius/freeagent/src/hestia/spec/hestia/message_multi_verifier_spec.rb:97:in `block (3 levels) in <module:Hestia>'
```

**Comment by caius** [comment]
#12 solves this!

**Comment by caius** [comment]
v0.3.0 has just been released on rubygems with support for Rails 5.0 & 5.1. Thanks to @afcapel @mauro-oto @etagwerker for their contributions!

cc @benlangfeld
